src/mainapp.cpp

```diff
@@ -418,11 +418,25 @@ int CMainApp::OnRun()
         countNinjas++;
     if (cNinja.CreateBuildFile(CNinja::GEN_RELEASE, CNinja::CMPLR_MSVC))
         countNinjas++;
+    if (cNinja.hasOptValue(OPT::TARGET_DIR32))
+    {
+        if (cNinja.CreateBuildFile(CNinja::GEN_DEBUG32, CNinja::CMPLR_MSVC))
+            countNinjas++;
+        if (cNinja.CreateBuildFile(CNinja::GEN_RELEASE32, CNinja::CMPLR_MSVC))
+            countNinjas++;
+    }
 #endif
     if (cNinja.CreateBuildFile(CNinja::GEN_DEBUG, CNinja::CMPLR_CLANG))
         countNinjas++;
     if (cNinja.CreateBuildFile(CNinja::GEN_RELEASE, CNinja::CMPLR_CLANG))
         countNinjas++;
+    if (cNinja.hasOptValue(OPT::TARGET_DIR32))
+    {
+        if (cNinja.CreateBuildFile(CNinja::GEN_DEBUG32, CNinja::CMPLR_CLANG))
+            countNinjas++;
+    if (cNinja.CreateBuildFile(CNinja::GEN_RELEASE32, CNinja::CMPLR_CLANG))
```

Comment:
A reminder to always run clang-format before checking in to avoid formatting glitches like this one.